docs/tutorial.rst

```diff
@@ -242,5 +242,151 @@ effective population size from the **TODO add crossref to catalog section**
 .. :ref:`sec_catalog_homo_sapiens_genome`
 
 
+************************************************
+Analysis: Genetic divergence between populations
```

Comment:
This is quite long for a heading --- how does it look in the sidebar?

Reply:
seems fine to me

Reply:
yeah, I was thinking this too. 
I think it would help to streamline things if we have a better way of organising these tutorials, especially now that there are a few more in the works. I think the most important distinction is between the tutorials that show people how to navigate the `stdpopsim` CLI/API, and the tutorials that show examples of analyses that can be performed on `stdpopsim` simulations using external software (like this one does). 

In the latest commit, I've restructured some of the headings to reflect this -- lmk what you think

Reply:
![Screen Shot 2019-12-05 at 12 34 36 pm](https://user-images.githubusercontent.com/17755074/70196069-a8391f00-175b-11ea-8f38-16910ad948f4.png)

Reply:
I've changed 'Sample' to 'Example', but otherwise this is what it looks like when you compile it

Reply:
Looks great, thanks.

Reply:
Yes, I was thinking the same thing. I think that organization works.